Core/CoreTests/Database/Models/DiscussionTopicTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import XCTest
+@testable import Core
+
+class DiscussionTopicTests: CoreTestCase {
+    func testProperties() {
+        let topic = DiscussionTopic.make()
+
+        XCTAssertEqual(topic.id, "1")
+        XCTAssertEqual(topic.title, "Graded Discussion")
+    }
+
+    func testSave() {
+        let api = APIDiscussionTopic.make()
+        DiscussionTopic.save(api, in: databaseClient)
```

Comment:
This test doesn't assert anything.. Maybe test that 1 discussion got saved?

Reply:
it's for coverage 😢